src/Tintin/Html/Templating.hs

```diff
@@ -213,14 +213,14 @@ tintinHeader info@Project.Info {..} Project.Page {..} =
           ]
     link_ [ rel_ "shortcut icon"
           , href_ ( asset $ "favicon-"
-                  <> bgColorOf info
+                  <> bgColorNameOf info
                   <> ".ico"
                   )
           ]
     link_ [ rel_ "stylesheet"
           , href_ "https://cdn.jsdelivr.net/npm/katex@0.10.0-alpha/dist/katex.min.css"
           ]
-    style_ Style.style
+    style_ $ Style.style $ Project.color info
```

Comment:
It is generally a good idea to use first function composition and in the end the application:

```suggestion
    style_ . Style.style $ Project.color info
```